Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -649,6 +649,16 @@ final class UIViewExtensionsTests: XCTestCase {
         // simple empty case test
         XCTAssertNil(container.widthConstraint)
     }
+
+    func testRemoveBlur() {
+        let view = UIView()
+        view.blur(withStyle: .prominent)
+        view.blur(withStyle: .extraLight)
+        view.removeBlur()
```

Comment:
Add a check before this that you have a blur

Reply:
Because it was my first contribution, there was some confusion. It's corrected.

Reply:
It's still not corrected. Just add a test after line 658 to check.

Reply:
It's a short code, but I'm sorry for making you review it multiple times. I modified the test code.